src/pages/certifications/front-end-libraries/react/create-a-component-with-composition/index.md

```diff
@@ -1,10 +1,26 @@
 ---
-title: Create a Component with Composition
+title:Create a Component with Composition
```

Comment:
Hi @angelobarbano 

The title of this page isn't rendering correctly. You need to add a space between `:` and the first letter of the title. Please see [this page](https://github.com/freeCodeCamp/guide/blob/654fa7b6aac26e10a2468b76d2f943e3b14705d5/src/pages/certifications/front-end-libraries/react/create-a-component-with-composition/index.md) to see how it's currently rendering.

Reply:
Thank you for your time reviewing my submission, I made the corrections.